R/01-mapPanelModule.R

```diff
@@ -0,0 +1,158 @@
+#' UI function of mapPanel module
+#'
+#' @param id module id
+#'
+#' @importFrom stats setNames
+#'
+mapPanelUI <- function(id) {
+  ns <- NS(id)
+  sidebarLayout(
+    sidebarPanel(
+      width = 2,
+      selectizeInputUI(
+        id = ns("group_name"),
+        label = "Group Name",
+        choices = unique(image_list$Group_name),
+        selected = NULL
+      ),
+      selectizeInputUI(
+        id = ns("variable"),
+        label = "Variable",
+        choices = NULL
+      ),
+      selectizeInputUI(
+        id = ns("measure"),
+        label = "Measure",
+        choices = NULL
+      ),
+      br(),
+      column(12,
+        align = "center",
+        radioButtonsUI(
+          id = ns("time_switch"),
+          choices = setNames(
+            c(1, 2),
+            c(
+              "Single Map",
+              "Time plot"
+            )
+          )
+        )
+      ),
+      # slider years are currently hardcoded and will be replaced later
+      sliderInputUI(
+        id = ns("time"),
+        label = "Time",
+        value = 2015
+      ),
+      # note: it is not possible to update a single value slider to range slider
+      # therefore we create two different sliders and toggle them based on the time_switch selection
+      shinyjs::hidden(
```

Comment:
as discussed, switch to a conditionalPanel later like in e.g.
https://github.com/Pandora-IsoMemo/plotr/blob/d8612fd4a3ca2978db06a572e14afcddc2737ed5/R/03-stylePlot.R#L88-L97